test/integration/dialects/postgres/query-generator.test.js

```diff
@@ -870,7 +873,7 @@ if (dialect.match(/^postgres/)) {
           expectation: 'CREATE INDEX \"bar\" ON \"User\" (\"username\", \"isAdmin\")'
         }, {
           arguments: ['mySchema.User', ['username', 'isAdmin'], {}, 'User'],
```

Comment:
Guess so.
I forgot what `quoteIdentifiers` does, does it split on `.`? If so yes that could probably work.